### DISCUSSION TITLE
How do i change my email

### DISCUSSION THREAD

**Discussion by omaribnzakaria** [community/community#127579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

whenever i sign into a microsoft product, it uses the email i used to create the github account, but i change my primary email to something else. is my email locked and i can only change my pulbic email or is there a secret setting

**Comment by BlakeGurnee** [comment]
You have to go into your GitHub settings by clicking on your profile in the upper right corner then scroll down and click settings. Once you have down this you can click where it says emails on the left of the page. After that, you can add an email set the email as your primary email and delete the other email.

**Reply by omaribnzakaria** [reply]
> You have to go into your GitHub settings by clicking on your profile in the upper right corner then scroll down and click settings. Once you have down this you can click where it says emails on the left of the page. After that, you can add an email set the email as your primary email and delete the other email.

i already did and it still uses the old email even though i deleted that off my account in the emails tab
